### PR TITLE
docs(llm): update configuration file related

### PR DESCRIPTION
**Pull request by returnToInnocence**
1. Add prompt configuration file description
2. Add a description of where the `.env` and prompt configuration files are located
3. The images use the schematics in the [hg-doc](https://github.com/apache/incubator-hugegraph-doc/tree/master/content/en/docs/images) repository  
    This is because the two documents are initially determined to be unified in content, so for the same pictures involved in the two documents, it is considered to use the pictures of the same warehouse for display, rather than maintaining two pictures in two different warehouses

### CODE REVIEW

File: hugegraph-llm/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # hugegraph-llm
 
-## Summary
+## 1 Summary
```
Comment (imbajin) [inline hugegraph-llm/README.md:3]:
```suggestion
## 1. Summary
```

update others also

File: hugegraph-llm/README.md [hunk 2]
```diff
@@ -15,25 +15,28 @@ graph systems and large language models.
...
+1. Start the HugeGraph database, you can run it via [Docker](https://hub.docker.com/r/hugegraph/hugegraph)/[Binary Package](https://hugegraph.apache.org/docs/download/download/).  
+    Refer to detailed [doc](https://hugegraph.apache.org/docs/quickstart/hugegraph-server/#31-use-docker-container-convenient-for-testdev) for more guidance
 
-1. Start the HugeGraph database, you can run it via [Docker](https://hub.docker.com/r/hugegraph/hugegraph)/[Binary Package](https://hugegraph.apache.org/docs/download/download/).
-Refer to detailed [doc](https://hugegraph.apache.org/docs/quickstart/hugegraph-server/#31-use-docker-container-convenient-for-testdev) for more guidance (PS: Graph visualization in step8)
 2. Clone this project
     ```bash
     git clone https://github.com/apache/incubator-hugegraph-ai.git
     ```
+
```
Comment (imbajin) [inline hugegraph-llm/README.md:32]: why need empty line for `1. 2. 3.`

**Review by returnToInnocence — COMMENTED** [review]
The above modification has been revised and restored

File: hugegraph-llm/README.md (hunk 2 above)
Comment (returnToInnocence) [reply hugegraph-llm/README.md:32]: Previously, I thought I was looking at unrendered markdown text. After all, line spacing is comfortable🧐

But forgetting to take the serial number into account will result in the wrong order, now correct it immediately🫡

File: hugegraph-llm/README.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ graph systems and large language models.
 2.  Use natural language to operate graph databases (Gremlin/Cypher)
 3.  Knowledge graph supplements answer context (GraphRAG)
 
-## Environment Requirements
+## 2 Environment Requirements
```
Comment (imbajin) [inline hugegraph-llm/README.md:18]:
```suggestion
## 2. Environment Requirements
```

File: hugegraph-llm/README.md [hunk 4]
```diff
@@ -15,15 +15,15 @@ graph systems and large language models.
...
 
-- python 3.9+ 
-- hugegraph-server 1.2+
+- python 3.9+  (better to use `3.10`)
+- hugegraph-server 1.3+
 
-## Preparation
+## 3 Preparation
```
Comment (imbajin) [inline hugegraph-llm/README.md:23]:
```suggestion
## 3. Preparation
```

File: hugegraph-llm/README.md [hunk 5]
```diff
@@ -90,9 +110,8 @@ This can be obtained from the `LLMs` class.
         .run()
     )
     ```
-
+    ![gradio-config](https://hugegraph.apache.org/docs/images/kg-uml.png)
```
Comment (imbajin) [inline hugegraph-llm/README.md:113]:
```suggestion

```

**Review by imbajin — APPROVED** [review]
TODO: update "gradio-config" picture in website repo

**Pull request merged**